examples/hpc-enterprise-slurm-v6.yaml

```diff
@@ -30,8 +30,6 @@ vars:
   # If image above is changed to use custom image, then setting below must be set to true
   instance_image_custom: false
   # When set, active compute nodes will be cleaned up on destroy.
-  # Note that setting this option requires additional dependencies to be installed locally.
-  # https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/schedmd-slurm-gcp-v6-controller#description
   enable_cleanup_compute: true
```

Comment:
For this blueprint, we want to keep `enable_cleanup_compute` variable and for hpc-slurm-static-v6 we dont  ?